app/javascript/components/CreateWork.jsx

```diff
@@ -12,22 +14,66 @@ class CreateWork extends React.Component {
       work: {
         title: '',
         material: '',
-        medium: 0,
-        availability: 0,
+        medium: "painting",
+        availability: "active",
         images: [],
+        featured_image: null,
         artist_id: this.props.artist_id
+      },
+      categories: {
+        "medium": 0,
+        "availability": 0
       }
     }
 
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
     this.onDrop = this.onDrop.bind(this);
+    this.mediumHandleChange = this.mediumHandleChange.bind(this);
+    this.availabilityHandleChange = this.availabilityHandleChange.bind(this);
+    this.selectFeaturedImage = this.selectFeaturedImage.bind(this);
+  }
+
+  componentDidMount() {
+    console.log(this.props.artist);
+    const route = APIRoutes.works.categories;
+    Requester.get(route).then(
+      response => {
+        this.setState({
+          categories: response,
+          componentDidMount: true
+        });
+        console.log(this.state.categories);
+      },
+      error => {
+        console.error(error);
+      }
+    );
   }
 
   handleChange(event) {
     const work = this.state.work;
+    console.log(event.target.name);
+    console.log(event.target.value);
     work[event.target.name] = event.target.value;
     this.setState({ work: work });
+    console.log(this.state.work);
+  }
+
+  mediumHandleChange(event) {
```

Comment:
I think that we can restructure this! you can create a function `handleChange(formAttr, value)` that sets the form attribute (ex. medium or availability) to the value. Then, we can just pass in this function into the onChange prop for each dropdown component

Reply:
fixed!